PublicationsRouter/v2/sword-out/XWALK-EPrints.md

```diff
@@ -0,0 +1,139 @@
+# JPER Core Metadata to Dublin Core/RIOXX XML
```

Comment:
See comments to `XWALK-DSpace.md` above: the filename,  the title and description all need to change. Also see comments about table layout which may? be applicable.

This page describes the PubRouter RIOXXplus XML output format.